build.gradle

```diff
@@ -1,12 +1,14 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
-    ext.kotlin_version = "1.5.0"
+    ext {
+        agp_version = '7.3.1'
```

Comment:
If we are only using this on line 11, I think it is a bit overkill to extract into a variable. What do you think?